src/follower.py

```diff
@@ -9,6 +9,7 @@
 from tagrec import TagRecognition
 import numpy as np
 
+import time
```

Comment:
Remove time if you're not using it (it doesn't look like you are)

Reply:
removed